values/istio-operator/istio-operator-raw.gotmpl

```diff
@@ -149,11 +149,27 @@ resources:
                 memory: 80Mi
             {{- end }}
             autoInject: enabled
-          tracer:
-            zipkin:
-              address: jaeger-operator-jaeger-collector.jaeger:9411 
           useMCP: false
-        meshConfig: {{- $i.meshConfig | toYaml | nindent 10 }}
+        meshConfig:
+          defaultConfig:
+            gatewayTopology:
+              numTrustedProxies: 1
+          enableAutoMtls: true
+          extensionProviders:
+          - EnvoyFileAccessLogProvider:
+            name: envoy-access-logs
+            path: /dev/stdout
+          {{- if $i.tracing.enabled }}
+            logFormat: |
```

Comment:
Depending on traceSampling not every log will have the `TRACEPARENT`.
Have you tested that scenario?

Reply:
If tracing is not enabled, envoy will use the default log formatting (without TRACEPARENT)